ui/src/app/kvm/components/KVMActionFormWrapper/ComposeForm/InterfacesTable/SubnetSelect/_index.scss

```diff
@@ -0,0 +1,81 @@
+// Widths are based on a 20:15:15 ratio for subnet:fabric:vlan columns, 2.5rem
+// PXE column, 1rem table cell padding and .25rem select padding.
+@mixin KVMSubnetSelect {
+  .kvm-subnet-select {
+    display: block;
+    width: 100%;
+
+    .kvm-subnet-select__dropdown {
+      max-width: none;
+      min-width: 0;
+      width: calc(2.5 * (100% + 0.75rem) + 2.5rem);
```

Comment:
Honestly I tried at first but it got pretty tricky. Basically the equation is determined by the ratio of the widths of the subnet/fabric/vlan columns, minus the horizontal paddings, and plus the static size of the PXE column. I can try again and hopefully remove all hard-coded values in this file so that the width is automatically calculated from the table scss file

Reply:
Updated: https://github.com/canonical-web-and-design/maas-ui/pull/1449/files#diff-40d30c0dc735278998878728a7e02b5eR1